### PR TITLE
Reapply: Change accent colors example image to HTML

### PR DESCRIPTION
**Pull request by In1quity**

### CODE REVIEW

**Comment by In1quity** [comment]
I hope this one patch is ok.

**Comment by In1quity** [comment]
@Volker-E done

File: visual-style.html [hunk 1]
```diff
@@ -245,7 +245,44 @@ <h1 class="page__title">Visual Style</h1>
 						<p>Accent colors are used to emphasise actions and highlight key information. Blue is a natural choice in our context, where it has been the default color used for links, conveying the idea of action.</p>
 						<p>There are three shades provided for situations where you need a lighter (Accent90), regular (Accent50) or a darker (Accent10) version.</p>
 						<p>Accent50 provides a blue which is suitable to be used for text and as background. When used as link text  it provides sufficient contrast with black text to notice the difference. When used as background, it provides enough contrast with white text to keep the text readable.</p>
-						<img src="img/colors-accent.png" alt="Accent colors">
+	-					<div class="color-palette color-section">
```
Comment (Volker-E) [inline visual-style.html]: There's an unwanted `-` at the beginning of the line.

**Comment by In1quity** [comment]
> The indentation is wrong by two tabs for the whole section.
two tabs for the whole section?

File: visual-style.html (hunk 1 above)
Comment (In1quity) [inline visual-style.html]: done

**Comment by Volker-E** [comment]
@In1quity Sorry, that is not coming from your patch set, it's a false leftover from earlier on.

**Pull request merged**